packages/collector/test/tracing/opentelemetry/fs-app.js

```diff
@@ -12,9 +12,17 @@ process.on('SIGTERM', () => {
   process.exit(0);
 });
 
+const expect = require('chai').expect;
+
 require('@instana/core/test/test_util/loadExpressV4');
 
-require('../../../src')();
+/**
+ * We install the latest version of the collector here locally.
+ * This ensures we are using the Opentelemetry production dependencies.
+ */
+expect(require.resolve('@instana/collector')).to.contain('opentelemetry/node_modules/@instana/collector');
```

Comment:
This ensures that we require the local dependency in packages/collector/test/tracing/opentelemetry/node_modules.